components/dashboard/table/Filters.js

```diff
@@ -68,10 +68,10 @@ export default function Filters({
             style={{
               position: strategy,
               top: y ?? '',
-              left: x ?? '',
+              left: x>0? x??'':"14px",
             }}
           >
-            <div className="bg-white w-96 shadow-xl rounded">
+            <div className="bg-white shadow-xl rounded w-[19rem]">
```

Comment:
use max-width property to make it responsive, don't hardcode the width of the component in both mobile and desktop views.